ui/src/message_popup/message_popup_page.jsx

```diff
@@ -138,26 +140,7 @@ export default React.createClass({
         <p style={styles.paragraph}>You may be asked to write, sketch or say your responses aloud.</p>
         <p style={styles.paragraph}>Each question is timed to simulate responding in the moment in the classroom.  You'll have 60 seconds to respond to each question.</p>
         <Divider />
-        <div style={{}}>
-          <div style={{fontSize: 16, padding: 20}}>
-            <Toggle
-              label="With student cards"
-              labelPosition="right"
-              toggled={this.state.shouldShowStudentCards}
-              onToggle={this.onStudentCardsToggled} />
-            <Toggle
-              label="With feeback and revision"
-              labelPosition="right"
-              toggled={false}
-              disabled={true}  />
-            <Toggle
-              label="With hints available"
-              labelPosition="right"
-              toggled={false}
-              disabled={true} />
```

Comment:
Yeah this is great!  I'll merge this now, and then let's go through the conflicts together.